NSS-Day-Cohort-42/holiday-road-eccentric-eels
# Filtered the weather to show only one timestamp per day. Included the days of the week in weather HTML.

# Steps to Review

1. `git fetch --all`
1. `git checkout mp-weather-forecast`
1. Verify that the forecast is filtered to display only one forecast for each day after a park is selected. 
1. Verify that the coding logic for the weather forecast is sound.
